src/processing/shape/LineDrawer.java

```diff
@@ -2,6 +2,10 @@
 
 import java.util.ArrayList;
 import processing.utility.*;
+import infrastructure.validation.logger.LoggerFactory;
+import infrastructure.validation.logger.ILogger;
+import infrastructure.validation.logger.LogLevel;
+import infrastructure.validation.logger.ModuleID;
```

Comment:
@reviewer??

Reply:
Reviewer should be me. It's Ahmed file, I will tell him to add